src/web/WebSchedulerService.cpp

```diff
@@ -367,26 +364,66 @@ bool WebSchedulerService::command(const char * cmd, const char * data) {
     return false;
 }
 
+#include "shuntingYard.hpp"
+
+bool WebSchedulerService::onChange(const char * cmd) {
+    for (const ScheduleItem & scheduleItem : *scheduleItems_) {
+        if (scheduleItem.active && scheduleItem.flags == SCHEDULEFLAG_SCHEDULE_ONCHANGE && Helpers::toLower(scheduleItem.time) == Helpers::toLower(cmd)) {
+#ifdef EMESESP_DEBUG
+            // emsesp::EMSESP::logger().debug(scheduleItem.cmd.c_str());
+#endif
+            return command(scheduleItem.cmd.c_str(), compute(scheduleItem.value.c_str()).c_str());
```

Comment:
compute() takes std::string so no need to convert to char *

Reply:
relict from first implementation, will change.